cosineSimilarity/Cosine.py

```diff
@@ -0,0 +1,23 @@
+from sklearn.feature_extraction.text import TfidfVectorizer
+from nltk.corpus import stopwords
+import numpy as np
+import numpy.linalg as LA
+
+cosine_function = lambda a, b : round(np.inner(a, b)/(LA.norm(a)*LA.norm(b)), 3)
+
+documents = ["The sky is blue.", "The sun is bright."]
+query = ["The sun in the sky is bright."]
+stopWords = stopwords.words('english')
+
+#vectorizer = TfidfVectorizer(stop_words = stopWords)
+vectorizer = TfidfVectorizer()
```

Comment:
We should specify stop words, stemmers, etc that get put in to the Vectorizer so it is consistent across all methods. I'll create an issue

Reply:
ahh i see. yea, this is the very basic version for now. the thing is: I don't need Sheetal and Jenna to preprocess anymore. I just need them to scrape out all the HTML stuff and give me a "content" text STRING. The TFIDFvectorizer from sklearn will do everything automatically for us.

Reply:
Right, but don't you have the ability to specify  the arguments I specified? It will use whatever you input as arguments